server/schema.ts

```diff
@@ -38,6 +38,12 @@ export namespace IConfig {
 		cookieSecureOnly: boolean;
 		mongoURL: string;
 		passwordResetExpiration: number;
+		services: {
+			auth: {
```

Comment:
Instead of the `| undefined`, you can make this an optional field by appending a `?` to `auth`. (i.e. the line becomes `auth?: {`)